source/draw/gpu/GpuYuv420Semiplanar.ooc

```diff
@@ -85,5 +85,21 @@ GpuYuv420Semiplanar: class extends GpuImage {
 		this y fill(ColorRgba new(yuv y, 0, 0, 255))
 		this uv fill(ColorRgba new(yuv u, yuv v, 0, 255))
 	}
+	_toRgba: func (target: GpuImage) {
+		shader := this _context getYuvToRgba()
+		shader add("texture0", this y)
+		shader add("texture1", this uv)
+		DrawState new(target) setMap(shader) draw()
+	}
```

Comment:
Is there perhaps some other name to help distinguish this from `toRgba`, besides the underscore?